src/components/Process/View.tsx

```diff
@@ -71,6 +75,39 @@ export const ProcessView = () => {
     if (election?.status === ElectionStatus.RESULTS) setTabIndex(1)
```

Comment:
Hey this condition also inside brackets please. Avoid single line conditions when doing logic.